climate_scanner/trends_innovation_classifier/trends_innovation_classifier.py

```diff
@@ -30,12 +30,13 @@ def test(self):
         """
         pass
 
-    def predict(self):
+    def predict(self,text):
         """
         A function to predict using the classifier.
         :return:
         """
-        pass
+        output = {"string_prediction":["building","3-d printing"],"string_prob":[0.9,0.5]}
```

Comment:
Is this not missing the string indices?

Reply:
I think this doesn't have to pass string indices back - In the overall loop this is retrieved directly. - the final output will have string indices.